docs/test-ui-bundle.md

```diff
@@ -0,0 +1,23 @@
+# Testing unmerged docs-ui changes
+
+If you want to test an unmerged change made in the `docs-ui` reprository with full content, build as usual the 
+documentation with the antora command but add the option `--ui-bundle-url <path-to-your-local-ui-bundle>/ui-bundle.zip` to your command.
+The location for this bundle is in the `docs-ui`reprository in directory `build/`.
+More details how the create the `ui-bundle.zip` can be found in the `docs-ui`reprository.
+
+The following example assumes:
+
+- You are in the root directory of your local `docs`repository
+- The `docs-ui`repository directory is on the same directory level as `docs`
+- You have `ui-bundle.zip` manually created with `gulp pack` (for details see `docs-ui`)
+- You have a webserver pointing to the `public` directory to access the built documentation
+
+```
+antora --clean \
+       --pull \
+       --ui-bundle-url ../docs/build/ui-bundle.zip \
+       --url http://localhost:5000 \
+       site.yml
+```
+
+If the build returns without an error, open the documentation with your browser to see your changes.
```

Comment:
How about linking "open the documentation with your browser" to the relevant section of the documentation that shows how to start a locally running copy?

Reply:
We do not have a "one shot fits all" section for that as there are multiple posibilities. Just see the broken link checker stuff. That is the reason how I wrote it as I did including a item in the assumption list.

Reply:
To me, it's not completely clear, especially for newcomers.